### PR TITLE
CVSL-2329 Tweaking reporting instruction journey for HDC licences and adding integration test

### PR DESCRIPTION
**Pull request by SebNegahbanMoJ**

### CODE REVIEW

File: server/views/pages/create/initialMeetingTime.njk [hunk 1]
```diff
@@ -28,23 +28,28 @@
...
-                    {% if licence.isEligibleForEarlyRelease %}
-                        {{ govukWarningText({
-                            html: "This person is eligible for an earlier release. It may affect their initial appointment. Find more information <br> in the <a class='govuk-link' rel='noreferrer noopener' target='_blank' href="+ fridayReleasePolicy +">Friday or pre-bank holiday release policy framework (opens in new tab).</a>",
-                            iconFallbackText: "Warning"
-                        }) }}
-                    {% endif %}
-
-                    {% set dateTimeHtml %}
+                }) %}
+                    {% if licence.kind == "HDC"%}
```
Comment (andrewrlee) [inline server/views/pages/create/initialMeetingTime.njk:32]: **suggestion:** would be tempted to move these to macros? 
```
{% if licence.kind == "HDC"%}
   ... call hdc macro
{% else % }
   ... call non hdc macro
```

File: server/views/pages/create/initialMeetingContact.njk [hunk 2]
```diff
@@ -18,7 +18,7 @@
         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full">
-                {% if licence.isEligibleForEarlyRelease %}
+                {% if licence.isEligibleForEarlyRelease and not licence.kind == "HDC" %}
```
Comment (andrewrlee) [inline server/views/pages/create/initialMeetingContact.njk:21]: it'd be nice to get this check pushed into the API at somepoint

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: server/views/pages/create/initialMeetingContact.njk (hunk 2 above)
Comment (andrewrlee) [reply server/views/pages/create/initialMeetingContact.njk:21]: (I saw the other use of `isEligibleForEarlyRelease` and it hadn't got the extra checks - but I assume that's ok as it's in the `else` so not HDC.., things like that are why it's good to to centralise logic like this)

Comment (SebNegahbanMoJ) [reply server/views/pages/create/initialMeetingContact.njk:21]: Absolutely, my intention is to make `isEligibleForEarlyRelease` just return `false` for HDC licences, but I need to make sure there's no other places that the flag is used and that that's the behaviour UCD want before I commit to it!

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**